documentation/source-code-api/index.markdown

```diff
@@ -57,8 +61,6 @@ In addition, the following repos exist for testing:
 
 We use [Travis](https://travis-ci.org/ros-planning/) continous integration combined with the [moveit_ci](https://github.com/ros-planning/moveit_ci) for testing pull requests and overall code health. Travis status badges should be visible on the README.md of every MoveIt! repository.
 
-To see an overview of the activity for MoveIt! check our [Open HUB Project Summary](https://www.openhub.net/p/moveit).
```

Comment:
I have no idea what was on the Open HUB page/if it's still being used, but I think the site's just down for maintenance (site says "[we] expect to be back online in 24 hours.").

Reply:
it was breaking our builds on two PRs so i got impatient, but yea its back now, adding

Reply:
```suggestion
To see an overview of the activity for MoveIt! check our [Open HUB Project Summary](https://www.openhub.net/p/moveit).
```